src/item_pocket.cpp

```diff
@@ -1092,6 +1092,15 @@ bool item_pocket::watertight() const
     return data->watertight;
 }
 
+bool item_pocket::bigger_on_the_inside( const units::volume &container_volume ) const
+{
+    if( rigid() ) {
+        return max_contains_volume() > container_volume;
+    } else {
+        return data->magazine_well > container_volume && max_contains_volume() > container_volume;
```

Comment:
If `data->magazine_well > max_contains_volume()` that's also illogical (although not actually a problem, per se). If you complain on that in `check_consistency` separately then this code could skip the second condition.